fallout/gatling/gatling_timeseries_latency.yaml

```diff
@@ -0,0 +1,99 @@
+# master uses a Gatling DSE plugin based on 3.x
+# DSP-17972-unified-driver uses a Gatling DSE plugin based on the unified driver (>= 4.5.x)
+branch: "master"
+
+---
+
+ensemble:
+  observer:
+    node.count: 1
+    provisioner:
+      name: ctool
+      properties:
+        cloud.provider: nebula
+        cloud.instance.type: m3.large
+    configuration_manager:
+      - name: ctool_monitoring
+        properties:
+          graphite.create_server: true
+  server:
+    node.count: 3
+    provisioner:
+      name: ctool
+      properties:
+        cloud.provider: nebula
+        cloud.tenant: moonshot-v2
+        cloud.instance.type: ms2.small
+    configuration_manager:
+      - name: ctool
+        properties:
+          product.type: cassandra
+          product.install.type: tarball
+          product.version: "3.11.6"
+      - name: ctool_monitoring
+        properties:
+          components: OS,JVM,GRAPH
```

Comment:
Does `Graph` in this context mean "DSE Graph"? If so maybe we can remove it.

Reply:
However according to the docs [here](https://fallout.sjc.dsinternal.org/components/configurationmanagers/ctool_monitoring) maybe we should add `cassandra`? We are not really interested in the cluster metrics, but maybe it's a nice to have.

Reply:
This entry is here largely to make sure the C* servers aren't being overloaded as part of this test.  I think "OS" and "JVM" are probably enough to accomplish that goal for now so I agree "GRAPH" should be removed.  I wouldn't say we _need_ "CASSANDRA" at this point... that seems like something we could add in later to diagnose what's causing the overload (if/when we see it).